fiat-roles/src/main/java/com/netflix/spinnaker/fiat/config/LockConfig.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2018 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.fiat.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.netflix.spectator.api.Registry;
+import com.netflix.spinnaker.kork.jedis.RedisClientDelegate;
+import com.netflix.spinnaker.kork.jedis.lock.RedisLockManager;
+import com.netflix.spinnaker.kork.lock.LockManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.time.Clock;
+import java.util.Optional;
+
+@Configuration
+public class LockConfig {
+  @Bean
+  Clock clock() {
+    return Clock.systemDefaultZone();
+  }
+
+  @Bean
+  LockManager redisLockManager(Clock clock,
+                               Registry registry,
+                               ObjectMapper mapper,
+                               RedisClientDelegate redisClientDelegate) {
+    return new RedisLockManager(
+        null, // will fall back to running node name
+        clock,
+        registry,
+        mapper,
+        redisClientDelegate,
+        Optional.empty(),
```

Comment:
heartbeat and lease durations are left at their defaults ... I don't know that this has any particular affect ... @jeyrschabu?

Reply:
Default values are reasonable. The rule of thumb is to keep `heartbeatRateMillis` less than the lock ttl to avoid expiring locks. In Igor I had to watch metrics and tweak these values accordingly.